tests/Microsoft.Identity.Test.Unit/CacheTests/CacheSerializationTests.cs

```diff
@@ -834,7 +834,7 @@ private void AssertAccessTokenCacheItemsAreEqual(MsalAccessTokenCacheItem expect
             Assert.AreEqual(expected.GetKey().ToString(), actual.GetKey().ToString());
             CollectionAssert.AreEqual(expected.ScopeSet.ToList(), actual.ScopeSet.ToList(), nameof(actual.ScopeSet));
             Assert.AreEqual(expected.TenantId, actual.TenantId, nameof(actual.TenantId));
-            Assert.AreEqual(expected.UserAssertionHash, actual.UserAssertionHash, nameof(actual.UserAssertionHash));
```

Comment:
So the cache key is replacing the userAssertionHash for these tests?

Reply:
This is just renaming of a property to better reflect what it can represent. Previously it was only a hash, now it can be a hash or a user provided cache key - either way both act as cache keys.